pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participantfiles/ParticipantFilesUseCase.java

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.dsm.model.participantfiles;
+
+import java.net.URL;
+import java.util.Optional;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.exception.DownloadException;
+import org.broadinstitute.dsm.model.elastic.ESFile;
+import org.broadinstitute.dsm.model.elastic.search.ElasticSearchParticipantDto;
+import org.broadinstitute.dsm.route.participantfiles.SignedUrlResponse;
+import org.broadinstitute.dsm.service.FileDownloadService;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+
+@Slf4j
+@AllArgsConstructor
+public class ParticipantFilesUseCase {
+
+    private static String CLEAN = "CLEAN";
+    private String bucketName;
+    private String blobName;
+    private String ddpParticipantId;
+    private String fileGuid;
+    FileDownloadService fileDownloadService;
+    Optional<DDPInstanceDto> ddpInstanceDto;
+
+
+    public boolean isFileClean() {
+        ddpInstanceDto.orElseThrow();
```

Comment:
No need to put it in a separate line I assume. You can just do
```
String participantIndex = ddpInstanceDto.orElseThrow().getEsParticipantIndex();
```

Reply:
Yes good catch!! I'll make the fix